projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-color.service.ts

```diff
@@ -50,7 +46,7 @@ export class DataLayerColorService {
     return undefined;
   }
 
-  /** build a CSS linear gradient that includes colors from all datasets in the layer */
+  /* build a CSS linear gradient that includes colors from all datasets in the layer */
```

Comment:
The extra `*` in `/**` means this is a documentation comment. Tools like jsdoc/typedoc that generate API documentation from the source code will look for it. It should not be removed.